agent/timings.js

```diff
@@ -90,17 +90,28 @@ function recordLcp() {
 
 function updateLatestLcp(lcpEntry) {
   if (lcp) {
-    var previous = lcp[0]
+    var previous = lcp
     if (previous.size >= lcpEntry.size) {
       return
     }
   }
-  lcp = [lcpEntry, cls]
+  lcp = lcpEntry
 }
 
 function updateClsScore(clsEntry) {
   if (clsSupported) {
-    cls += clsEntry.value
+    var n = now()
```

Comment:
clsEntry.startTime might be better to use here, since it is guaranteed to be the exact time the shift occurred